applications/connectivity_bridge/src/disk/config.c

```diff
@@ -111,7 +111,7 @@ static struct cfg_option configs[] = {
 
 static const char file_contents_header[] =
 "==========================================\r\n"
```

Comment:
I suppose we should line up the border so the title is still centered
```suggestion
"=========================\r\n"
```